backend/routes/api/groups.js

```diff
@@ -38,16 +37,20 @@ router.get("/currentUser", requireAuth, async (req, res) => {
         }
     })
 
-    const groupId = await Membership.findOne({
+    const groupId = await Membership.findAll({
         where: {
-            memberId: currentUser.id
+            memberId: currentUser.id,
+            status: {
+                [Op.in]: ['co-host', 'member']
+            }
         }
     })
 
     if (groupId) {
```

Comment:
Since owned groups and joined groups are being queried separately and no error is expected. There would be no need for the condition for groupId.